add missing 4 to link to shoes4 rdocs.

nicholalexander opened the pull request:
fix issue #86 - link was missing '4'.

unless it's supposed to reference shoes3? 

http://www.rubydoc.info/github/shoes3/shoes3

nicholalexander committed:
add missing 4 to link to shoes4 rdocs.

jasonrclark commented:
So actually there should probably be two separate links: one to the shoes4 docs you found @nicholalexander and one to http://www.rubydoc.info/github/shoes/shoes3 (note the earlier `shoes` doesn't have a 3 anymore... moving things around recently).

Since both projects are active and have different internal structure, the docs will be valuable in both cases.

nicholalexander committed:
add multiple links for rdocs.

update shoes3 link.
uupdate labeling for both links.
use rubydoc.info to avoid redirects.

nicholalexander commented:
@jasonrclark updated!

jasonrclark commented:
✨

nicholalexander commented:
thanks!  shoes on!
